submissions/EnF1NeeT/DOM-task/script.js

```diff
@@ -0,0 +1,169 @@
+<<<<<<< HEAD
+const postsList  = [
+||||||| 76728da
+const postsLst = [
+=======
+const postsList = [
+>>>>>>> 2707486fe7b80d419dcc0d7f8c3d8c626d02b823
+    {
+        id: '1',
+        topzoneImg:'./img/2.png',
+        nickname:'@Jonnie',
+        imgSrc: './img/post1.jpg',
+        imgSrc2: './img/post2.jpg',
+        imgSrc3: './img/post3.jpg',
+        imgSrc4: './img/post4.jpg'
```

Comment:
If the user has more than thousands of photos in his collection. Then you create a new property with a photo. I recommend saving the collection of photos into an array.

Reply:
Done